docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -149,9 +152,77 @@ an expert; this is meant to be accessible for all people. Wherever you are in th
 journey, this guide is intended to be a clear, helpful resource.  
 
 We’re not going to put any words into other people’s mouths, but we will use our own voice to do 
-the work and use the terms we find most accurate and inclusive.  
+the work and use the terms we find most accurate and inclusive.
+
+1.2 Introduction to sex and gender
+----------------------------------
 
-1.2 Glossary of General Terminology
+Most human fetuses have either two X chromosomes or one X and one Y chromosome.
+A fetus' configuration of X and Y chromosomes influences a number of aspects of development,
+such as hormones, gonads, and genitalia.
+Then the child is born into a society with lots of pre-existing ideas about how bodies should
+look and work, how people should behave, and how all these things should relate to one another.
+Almost immediately, expectations and norms about identity and behavior are placed on the child,
+based on their external genitalia at birth.
+As that child grows up and throughout their life, societal pressures and norms impact their identity and behavior,
+and their identity, behavior, and experiences have impacts in turn,
+including on very "biological" factors such as hormone levels.
+Meanwhile, their chromosomal makeup, hormones, etc. continue to have evolving impacts on their body,
+which in turn can affect their identity and behavior, and how society views them.
+
+This constellation of attributes is what we mean by "sex and gender."
+It is an incredibly broad umbrella, spanning everything from
+chromosomes to clothing styles to power structures.
+Despite this range, it is difficult to tease apart the causal impacts of these factors from
+one other, because all these factors are strongly correlated with each other.
+
+The dominant culture in the West has historically seen sex and gender as synonymous and
+binary.
+According to the traditional view, a person is born either male or female,
+and people born male should identify and behave like boys and men,
+while people born female should identify and behave like girls and women.
```

Comment:
```suggestion
and it is generally assumed that people born male will identify and behave like boys and men,
while people born female will identify and behave like girls and women.
```
Slight preference to remove "should" here to be a bit more neutral, but curious if others have different opinions.

Reply:
Does this also apply to these lines above?

> Then the child is born into a society with lots of pre-existing ideas about how bodies should
look and work, how people should behave, and how all these things should relate to one another.

I think your edit highlights the ways that societal pressures are created by well-meaning people, which I like. I don't want to totally eliminate reference to the more intentional and conscious ways gender is enforced, though.

Reply:
True - I like how the word 'should' is a moral indicator - it signifies a negative consequence for those who do not identify/behave as expected.

Reply:
What is we put quotation marks around the word should just to make it more clear that we are not suggesting that this is how people actually "should" behave to the quick reader skimming through the section without full context -- I think that was most of my concern here.

Reply:
I like that! I'll do it in both places.